source/Symbol/SwiftASTContext.cpp

```diff
@@ -3009,9 +3010,17 @@ swift::ASTContext *SwiftASTContext::GetASTContext() {
   }
 
   // Install the parseable interface module loader.
-  std::string ModuleCachePath = GetClangImporterOptions().ModuleCachePath;
-  StringRef PrebuiltModuleCachePath =
-      GetCompilerInvocation().getFrontendOptions().PrebuiltModuleCachePath;
+  std::string moduleCachePath = GetClangImporterOptions().ModuleCachePath;
+
+  // Use the default location of the prebuilt module cache:
+  // <resource-dir>/<platform>/prebuilt-modules
+  llvm::Triple triple(GetTriple());
+  llvm::SmallString<64> prebuiltModuleCachePath = GetResourceDir(triple);
```

Comment:
Is 64 enough for a path deep inside Xcode.app?

Reply:
Owise we should either make it much smaller or large enough for /Applications/Xcode.app

Reply:
Good point. Bumped it up to 128 (the default Xcode toolchain case needs 115).